imports/plugins/included/product-variant/components/productGrid.js

```diff
@@ -96,23 +127,33 @@ class ProductGrid extends Component {
   }
 
   renderFilteredCount() {
-    const { selectedProductIds, classes } = this.props;
-    const count = selectedProductIds.length;
+    const { selectedProductIds, totalProductCount, classes } = this.props;
+    const selectedCount = selectedProductIds.length;
     const filterByProductIds = Session.get("filterByProductIds");
 
     if (filterByProductIds) {
       return (
         <div className={classes.filterCountContainer}>
-          <Typography variant="h4" display="inline">
+          <Typography variant="h4" display="inline" className={classes.productsTitle}>
             {i18next.t("admin.productTable.bulkActions.filteredProducts")}
           </Typography>
           <Typography variant="h5" display="inline" className={classes.filterCountText}>
-            {i18next.t("admin.productTable.bulkActions.selectedCount", { count })}
+            {i18next.t("admin.productTable.bulkActions.selectedCount", { selectedCount })}
           </Typography>
         </div>
       );
     }
-    return "";
+    const totalOrSelected = (selectedCount > 0) ? `${selectedCount} selected` : `${totalProductCount} products`;
```

Comment:
Should be translated with singular/plural forms like `admin.productTable.bulkActions.selectedCount`.